src/test/regress/multi_schedule

```diff
@@ -129,33 +127,17 @@ test: multi_outer_join
 # is independed from the rest of the group, it is added to increase parallelism.
 # ---
 test: multi_create_fdw
-test: multi_complex_count_distinct multi_select_distinct
+test: multi_complex_count_distinct multi_select_distinct multi_modifications
 test: multi_distribution_metadata
-test: multi_generate_ddl_commands
-test: multi_create_shards
-test: multi_prune_shard_list
-test: multi_repair_shards
-test: multi_modifications
-test: multi_upsert
-test: multi_simple_queries
-test: multi_create_insert_proxy
-test: multi_data_types
-test: multi_utilities
-test: multi_repartition_udt
-test: multi_repartitioned_subquery_udf
-test: multi_modifying_xacts
-test: multi_transaction_recovery
-test: multi_subtransactions
+test: multi_generate_ddl_commands multi_create_shards multi_prune_shard_list multi_repair_shards multi_upsert multi_simple_queries multi_create_insert_proxy multi_data_types
+test: multi_utilities multi_modifying_xacts
+test: multi_repartition_udt multi_repartitioned_subquery_udf multi_transaction_recovery multi_subtransactions
```

Comment:
multi_transaction_recovery tests rely on pg_dist_transaction table but this did not create any problem